styles.css

```diff
@@ -81,225 +297,255 @@ body {
-    line-height: 16px;
-    color: #7a869a;
-  }
-
-  .fiftin {
-    font-family: 'Poppins', Arial, Helvetica, sans-serif;
-    font-style: normal;
-    font-weight: 600;
-    font-size: 13px;
-    line-height: 16px;
-    color: #7a869a;
-  }
-
-  .art {
-    font-family: 'Poppins', Arial, Helvetica, sans-serif;
-    font-style: normal;
-    font-weight: 400;
+    font-weight: 500;
     font-size: 15px;
-    line-height: 24px;
+    line-height: 20px;
     color: #344563;
+    margin-right: 10vw;
+    text-decoration: none;
   }
 
-  .frontend {
-    font-family: 'Poppins', Arial, Helvetica, sans-serif;
-    font-style: normal;
-    font-weight: 500;
-    font-size: 12px;
-    line-height: 16px;
-    color: #6070ff;
-    list-style-type: none;
-    display: flex;
-    gap: 16px;
-    padding-left: 0;
-  }
-
-  .frontend li {
-    background: #ebebff;
-    padding: 4px 12px;
-    border-radius: 8px;
-  }
-
-  .seeproject {
-    font-family: 'Poppins', Arial, Helvetica, sans-serif;
-    font-style: normal;
-    font-weight: 500;
-    font-size: 17px;
-    line-height: 24px;
-    color: #396df2;
-    border: 1px solid #6070ff;
-    border-radius: 8px;
-    background-color: inherit;
-    padding: 12px;
-    gap: 10px;
-    margin-top: 16px;
-    margin-bottom: 40px;
+  nav {
+    display: none;
   }
 
   .card1 {
     display: flex;
-    flex-direction: column;
-    justify-content: center;
+    flex-direction: row;
+    justify-content: space-around;
     padding: 16px;
     border-radius: 16px;
     border: solid #e5e5e5 1px;
     background-color: #fff;
+    gap: 24px;
   }
 
   .about {
+    display: flex;
     background-color: #fff;
     border-radius: 0 100px 0 0 / 100px;
     padding-top: 8%;
     margin-top: 16%;
   }
 
   #about {
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
     margin: 4vw;
   }
 
-  .aboutmyself {
-    margin-left: 8vw;
-    margin-right: 8vw;
+  #prlanguages {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    margin-right: 14vw;
+  }
+
+  #prlanguages li {
+    display: flex;
+    flex-direction: column;
   }
 
-  .separ {
-    margin-right: 24px;
+  .languages li h4 {
+    color: #344563;
   }
 
-  .languages {
+  .contactform {
     display: flex;
-    justify-content: space-between;
+    flex-direction: column;
     align-items: center;
+    margin-top: 50px;
+    gap: 50px;
+    background: #6070ff;
+    background-image: url(icons/contact\ form\ background\ shapes@2x.png);
+    background-size: cover;
+    border-radius: 100px 0 0 0 / 100px;
+  }
+
+  .submitted {
+    cursor: pointer;
+    padding: 12px 16px;
+    width: 142px;
+    height: 48px;
+    border: 1px solid #cfd8dc;
+    background: #fff;
+    border-radius: 8px;
+    color: #6070ff;
+    letter-spacing: 0.03em;
     font-family: 'Poppins', Arial, Helvetica, sans-serif;
     font-style: normal;
     font-weight: 500;
-    font-size: 20px;
+    font-size: 17px;
     line-height: 24px;
```

Comment:
- [ ] Please 🙏🏽 fix the text in the submit button to be similar to the Figma template 👇🏽 

| Figma template | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/220825415-14892dc2-5718-47fb-9597-f43bfc51fe3c.png) | ![image](https://user-images.githubusercontent.com/58642949/220825465-2e594cc0-d37a-46b6-a1e5-d2c846963a09.png) |